srcv2/App.js

```diff
@@ -3,16 +3,19 @@ import { withRouter, Redirect } from 'react-router-dom'
 import PublicRoute from './components/PublicRoute'
 import PrivateRoute from './components/PrivateRoute'
 
-import Footer from './components/Footer'
+import Authentication from './components/Authentication'
 
 import DashboardPage from './pages/DashboardPage'
 import LandingPage from './pages/LandingPage'
 import LoginPage from './pages/LoginPage'
+import OnboardingPage from './pages/OnboardingPage'
+import OnboardingContributorPage from './pages/OnboardingContributorPage'
 
 class App extends React.Component {
     render() {
         return (
             <div className='App'>
+                <Authentication/>
                 <div className='content pb-24'>
                     <PublicRoute
                         path='/'
```

Comment:
Make the `LandingPage` and `LoginPage` routes restricted
```suggestion
                <div className='content pb-24'>
                    <PublicRoute
                        restricted
                        path='/'
```